metadata-ingestion/cli-ingestion.md

```diff
@@ -1,66 +1,140 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # CLI Ingestion
 
-## Installing the CLI
+Batch ingestion involves extracting metadata from a source system in bulk. Typically, this happens on a predefined schedule using the [Metadata Ingestion ](metadata-ingestion/README.md#install-from-pypi)framework.
+The metadata that is extracted includes point-in-time instances of dataset, chart, dashboard, pipeline, user, group, usage, and task metadata.
 
-Make sure you have installed DataHub CLI before following this guide.
+## Installing DataHub CLI
 
-```shell
-# Requires Python 3.8+
+:::note Required Python Version
+Installing DataHub CLI requires Python 3.6+.
+:::
+
+<Tabs>
+<TabItem value="pypi" label="Install from PyPI for OSS Release" default>
```

Comment:
@hsheth2 let me know if making this as tabs makes sense

Reply:
actually this whole gemfury thing is outdated and can be removed